### ISSUE TITLE
Possible [BUG] 

### ISSUE THREAD

**Issue by bigpoppapaul** [nixgates/plugin.video.seren#871]
Running Seren on a Shield with 19.4 and AH skin. 

Finding that with Seren 3.0, the Next Up Dialogue only seems to work for one episode. Meaning if I start a show, the Next Up will appear for the following episode but NOT the episode after that.  I have Next Up set to go to "source selection," but even that screen doesn't show up if I watch two eps in a row, it just goes back to the skin. 

Is there a new setting I missed? If you're not able to reproduce the bug let me know and I'll dig around my own system to see if anything is messed up. 

Thanks.

**Comment by nixgates** [comment]
A log would be lovely here if you could.

Sounds like something in background scraping may be failing. I can't reproduce this on my end as Seren is able to play for hours continually for myself. It might be settings specific.

**Comment by bigpoppapaul** [comment]
> A log would be lovely here if you could.
> 
> Sounds like something in background scraping may be failing. I can't reproduce this on my end as Seren is able to play for hours continually for myself. It might be settings specific.

Sorry about that. I definitely should have included a log, I just wasn't thinking. I'll have one for you as soon as the insanity of the next couple days is done. 
Happy New Year man, thank you for all you do.

**Comment by cgwprs** [comment]
I have seen that issue as well. I will try to get a log to you soon. For me the show that was playing would work for about 3 episodes and then stop. Xubuntu 20.04, Kodi 19.5, Seren 3.0.0 latest a4kscrapers.

**Comment by cgwprs** [comment]
[kodi.log](https://github.com/nixgates/plugin.video.seren/files/10328055/kodi.log)
There are a few lines of "Python interpreter stopped" near the end of the log. Please let me know if you need additional information.

**Comment by nixgates** [comment]
@cgwprs @bigpoppapaul 

Just to confirm that there is more items in the playlist to be played still? As dialogue will only show up in this instance. 

Looking at your logs @cgwprs I can see the call to open up the dialogue did run multiple times.

There is another instance where it won't work, if the background scrape can not find a source for the next item in the playlist, it will clear out the playlist and playback will stop after the current item has finished. Because the playlist is empty, no dialogue will show as well.

Unfortjnately I currently can't reproduce this on any of my devices.

**Comment by bigpoppapaul** [comment]
> @cgwprs @bigpoppapaul
> 
> Just to confirm that there is more items in the playlist to be played still? As dialogue will only show up in this instance.
> 
> Looking at your logs @cgwprs I can see the call to open up the dialogue did run multiple times.

@nixgates 

Yes, there were further episodes available. Once I played it again through Seren, it scraped the episode fully and I went on to watch it. 
I was also wondering if it may have been an isolated incident with that one show, but unfortunately, tonight as we were watching something else with 3 episodes available, it happened again. The Next Up dialogue popped up for the second episode, but then not for the the 3rd. 

[Here is the log file I pulled after it happened](https://pastebin.com/xKUsX4Lt)

**Comment by vermope** [comment]
Just to say that I experience the same thing. Seren is invoked via TMDH Helper player.

**Comment by bigpoppapaul** [comment]
> Just to say that I experience the same thing. Seren is invoked via TMDH Helper player.
> 
> When the first episode is played, I see the created playlist in my 'remote'. After the subsequent episode, there is no playlist to be seen. [kodi.log](https://github.com/nixgates/plugin.video.seren/files/10519158/kodi.log)

Yep, I also have Seren invoked through TMDBHelper. I wonder if that's the common thread for all of us.

**Comment by welovewebs** [comment]
First of all, let me say thank you for such a well designed plugin!
And also apologize for the lack of a log at this time.

This is also happening to me. Fire Stick 4k Kodi 19.5 and 20.1 TMDBHelper and Arctic Horizon 2 skin
Also Fire Cube latest Kodi TMDB and AH2.
Always plays one additional episode then stops.

I have just got the Fire Cube, so when I get ADB working, I'll grab the log.
